packages/foam-vscode/src/core/model/workspace.test.ts

```diff
@@ -447,7 +500,7 @@ describe('Wikilinks', () => {
   });
 });
 
-describe('markdown direct links', () => {
+describe('Markdown direct links', () => {
```

Comment:
I've capitalized a couple of descriptions because most of the them start with a capital letter.